stores/appStore.ts

```diff
@@ -86,6 +92,23 @@ export const useAppStore = defineStore("app", {
       return state.currentComparison.scenarios?.length > 0
         && state.currentComparison.scenarios?.every(s => !!s.runId);
     },
+    everyScenarioHasCosts: (state): boolean => {
+      return state.currentComparison.scenarios?.map(s => s.result.data?.costs).every(c => !!c && c.length > 0);
+    },
+    axisLabel(state): string | undefined {
+      if (state.currentComparison.axis) {
+        return this.parametersMetadataById[state.currentComparison.axis].label;
+      }
+    },
+    baselineScenario(state): Scenario | undefined {
+      return state.currentComparison.scenarios.find((scenario) => {
+        if (state.currentComparison.axis && scenario.parameters) {
```

Comment:
I guess you could return undefined from the top level if !state.currentComparison.axis, rather than getting into the find at all. 

Why would scenario.parameters be falsy?

Reply:
> I guess you could return undefined from the top level if !state.currentComparison.axis, rather than getting into the find at all.

True, but the TS linter objects to that, for some reason (`Type 'undefined' cannot be used as an index type`), whereas it doesn't when the check is closer to the usage (`scenario.parameters[state.currentComparison.axis]`).

> Why would scenario.parameters be falsy?

The type signature for the property is `ParameterSet | undefined`. A specific situation where it might be undefined is in the last few lines of appStore.loadScenarioDetails : if the request for the scenario fails, we don't assign any parameters to the scenario.